ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/network/P2PNetworkTest.java

```diff
@@ -269,71 +258,12 @@ public void rejectIncomingConnectionFromBlacklistedPeer() throws Exception {
     }
   }
 
-  @Test
-  public void rejectIncomingConnectionFromNonWhitelistedPeer() throws Exception {
```

Comment:
This case is covered more generically by the test `rejectIncomingConnectionFromDisallowedPeer`.